### PR TITLE
Fix a-vs-an typos

### PR DESCRIPTION
**Pull request by practicalswift**
Fix a-vs-an typos.

### CODE REVIEW

File: src/script/ismine.h [hunk 1]
```diff
@@ -29,7 +29,7 @@ enum isminetype
 typedef uint8_t isminefilter;
 
 /* isInvalid becomes true when the script is found invalid by consensus or policy. This will terminate the recursion
- * and return a ISMINE_NO immediately, as an invalid script should never be considered as "mine". This is needed as
+ * and return an ISMINE_NO immediately, as an invalid script should never be considered as "mine". This is needed as
```
Comment (promag) [inline src/script/ismine.h]: Remove `an`?

**Review by promag — COMMENTED** [review]
ACK.

File: src/script/ismine.h (hunk 1 above)
Comment (practicalswift) [reply src/script/ismine.h]: Thanks! Fixed!

**Comment by randolf** [comment]
ACK for 11 out of 12 changes...

NACK for 1 change in src/reverselock.h because the original "An RAII-style" is actually correct since acronyms tend to be read out letter-by-letter and the word "An" flows more easily and correctly when reading out loud.

**Review by randolf — CHANGES_REQUESTED** [review]
ACK for 11 out of 12 changes...

NACK for 1 change in src/reverselock.h because the original "An RAII-style" is actually correct since acronyms tend to be read out letter-by-letter and the word "An" flows more easily and correctly when reading out loud.

**Comment by practicalswift** [comment]
@randolf Good point about RAII. Fixed. Please re-review :-)

**Comment by randolf** [comment]
@practicalswift I don't see the fix yet, but perhaps GitHub just needs some time to synchronize.  I'll check again later and approve once the fix appears.  Thanks for the quick turn-around.

**Comment by randolf** [comment]
@practicalswift I just asked some other developers about this, and they suspect that you may need to "push" your changes so that they'll appear here.

**Comment by practicalswift** [comment]
@randolf Oh, I forgot to push after changing. Thanks for letting me know! Please re-review :-)

**Review by randolf — APPROVED** [review]

**Comment by MarcoFalke** [comment]
Travis fails with `FAIL: subtree directory was touched without subtree merge`

**Comment by practicalswift** [comment]
Removed the change to `src/univalue/` since that one should be reported upstream instead.

Please re-review :-)

**Review by randolf — APPROVED** [review]

**Review by luke-jr — APPROVED** [review]
utACK

**Pull request merged**